### ISSUE TITLE
Generic Builder should not need explicit type for build method

### ISSUE THREAD

**Issue by violplayer** [mplushnikov/lombok-intellij-plugin#249]
The following code shows a warning about usage of raw types when build() is called although the type of the builder b is well defined as Test.TestBuilder<String>. 

```
import lombok.Builder;
import lombok.Value;

@Builder
@Value
public class Test<T> {
    T value;

    public static void main(String[] args) {
        final TestBuilder<String> b = Test.<String>builder().value("a");
        final Test<String> t = b.build();
    }
}
```

This warning can only be removed when replacing with

```
final Test<String> t = b.<String>build();
```

**Comment by AlexejK** [comment]
Just out of curiosity - is this actually working in Eclipse and supported by native lombok?

**Comment by violplayer** [comment]
![screen](https://cloud.githubusercontent.com/assets/1357931/16867769/0506382a-4a74-11e6-9b75-27197d8b730a.JPG)

Native lombok doesn't know about generic type because of type erasure. I just added a screenshot that shows that at least in the structure, you can see the build() method return Data<T> and does not need static type added to the call.

**Comment by AlexejK** [comment]
We will discuss this and see if this actually something we should support.
The goal is to mimic Eclipse functionality (supported by Lombok natively) as closely as possible.

**Comment by mplushnikov** [comment]
Hello @violplayer, sorry for the huge delay!
I think this issue was already fixed in the last release of the plugin (0.13). It is not reproducible for me any more.

**Comment by violplayer** [comment]
Hello @mplushnikov,
thanks for your effort. Seems to be fixed in version 0.13.14.

Best regards
Sven
